content/services/services_powerdev.rst

```diff
@@ -4,6 +4,9 @@ PowerLinux / OpenPOWER Development Hosting
 :title: Development Hosting
 :menu: Services, Development Hosting, 3
 
+.. |br| raw:: html
+
+    <br />
```

Comment:
Is there a particular reason why we needed to add this? Can't this be fixed in css or at least have a class we can call using ``rst-class`` instead of using raw html?